Source/Core/DiscIO/WbfsBlob.cpp

```diff
@@ -40,7 +40,7 @@ WbfsFileReader::WbfsFileReader(const std::string& filename)
 
 WbfsFileReader::~WbfsFileReader()
 {
-	for (u32 i = 0; i != m_files.size(); ++ i)
+	for (u32 i = 0; i != m_files.size(); ++i)
```

Comment:
for (auto& f : m_files[i]) delete f;

Or better, make that a vector/array of unique_ptrs, if possible.